osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/CheckK8sSecurityGroupLabelMetaTask.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.tasks.conformance.openstack.securitygroup;
+
+import javax.persistence.EntityManager;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.osc.core.broker.job.Task;
+import org.osc.core.broker.job.TaskGraph;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
+import org.osc.core.broker.service.tasks.TransactionalMetaTask;
+import org.osgi.service.component.annotations.Component;
+
+@Component(service = CheckK8sSecurityGroupLabelMetaTask.class)
+public class CheckK8sSecurityGroupLabelMetaTask extends TransactionalMetaTask {
+
+    private SecurityGroupMember sgm;
+    private boolean isDelete;
+
+    @Override
+    public TaskGraph getTaskGraph() {
+        // TODO unimplemented
+        throw new NotImplementedException(getClass());
+    }
+
+    @Override
+    public String getName() {
+        // TODO Auto-generated method stub
+        return "Check Kubernetes Security Group Label Meta Task -- unimplemented";
```

Comment:
@dmitryintel I believe the reason why the multi-member test is failing it because of this... Every task **instance** in the graph must have a unique name. This means that the string returned by getName but be unique in the graph. For that ensure to include the entity id in the string, in this case the sgm id also ensure that this string "Check k8s security group label..." is not being used in any other get name. Note: I had an issue where I was using the id but had the same description copied from another task. This should solve ur multi member test issue.